plugins/org.erlide.help/articles/eclipse/170_FAQ.md

```diff
@@ -11,7 +11,7 @@ part: Getting started
 ## I can't write comments or strings in Chinese (or other non-latin language)!
 <a name='faq1'/>
 
-This has been fixed for R1603+. We use the right default encoding according to the runtima and we detect the `coding` comments at the top of a file and use the right encoding for it. There might be issues if you have multiple projects in the workspace targeting different Erlang versions, use `coding` to fix them.
+This has been fixed for R1603+. We use the right default encoding according to the runtime and we detect the `coding` comments at the top of a file and use the right encoding for it. There might be issues if you have multiple projects in the workspace targeting different Erlang versions, use `coding` to fix them.
```

Comment:
R16 is so old so maybe we can delete this section?

Reply:
Removed mentioning of R16 and added some links to `coding` if anyone gets the problem.